astroquery/jplhorizons/core.py

```diff
@@ -1060,18 +1075,17 @@ def vectors_async(self, *, get_query_payload=False,
                 commandline += ' CAP{:s};'.format(closest_apparition)
             if no_fragments:
                 commandline += ' NOFRAG;'
-
-        if isinstance(self.location, dict):
-            raise ValueError(('cannot use topographic position in state'
-                              'vectors query'))
+        #
+        # if isinstance(self.location, dict):
+        #     raise ValueError(('cannot use topographic position in state'
+        #                       'vectors query'))
```

Comment:
Do remove unused code rather than comment out

Reply:
oops. missed this.